src/um2netcdf.py

```diff
@@ -347,52 +403,57 @@ def keyfunc(c):
                 print('\nMissing lat/lon coordinates for variable (possible timeseries?)\n')
                 print(c)
                 raise Exception("Variable can not be processed")
+
             fix_level_coord(c, z_rho, z_theta)
 
             if not args.nomask and stashcode.section == 30 and \
-             (201 <= stashcode.item <= 288  or 302 <= stashcode.item <= 303):
+                    (201 <= stashcode.item <= 288 or 302 <= stashcode.item <= 303):
                 # Pressure level data should be masked
                 if have_heaviside_uv:
-                    apply_mask(c, heaviside_uv, args.hcrit)
+                    apply_mask(c, heaviside_uv, args.hcrit)  # noqa TODO: fix referencing warning
                 else:
                     continue
-            if not args.nomask and stashcode.section == 30 and \
-             (293 <= stashcode.item <= 298):
+
+            if not args.nomask and stashcode.section == 30 and (293 <= stashcode.item <= 298):
                 # Pressure level data should be masked
                 if have_heaviside_t:
-                    apply_mask(c, heaviside_t, args.hcrit)
+                    apply_mask(c, heaviside_t, args.hcrit)  # noqa TODO: fix referencing warning
                 else:
                     continue
+
             if args.verbose:
                 print(c.name(), itemcode)
+
             cubewrite(c, sman, args.compression, args.use64bit, args.verbose)
 
+
 if __name__ == '__main__':
-    import sys, argparse
     parser = argparse.ArgumentParser(description="Convert UM fieldsfile to netcdf")
     parser.add_argument('-k', dest='nckind', required=False, type=int,
-                        default=3, help='specify kind of netCDF format for output file: 1 classic, 2 64-bit offset, 3 netCDF-4, 4 netCDF-4 classic model. Default 3', choices=[1,2,3,4])
+                        default=3,
+                        help='specify netCDF output format: 1 classic, 2 64-bit offset, 3 netCDF-4, 4 netCDF-4 classic model. Default 3',
```

Comment:
I think this line will get PEP8 errors for line too long

Reply:
Yep, I had the error but decided to defer string fixes to another issue/PR.